Detectors/TPC/reconstruction/src/TPCCATracking.cxx

```diff
@@ -109,8 +111,9 @@ int TPCCATracking::runTracking(TChain* inputClustersChain, const TClonesArray* i
         mClusterData[i].Allocate(mClusterData[i].NumberOfClusters() + nClusters[i]);
       }
     }
-    for (Int_t icluster = 0; icluster < inputClustersArray->GetEntries(); ++icluster) {
-      Cluster& cluster = *static_cast<Cluster*>(inputClustersArray->At(icluster));
+    for (Int_t icluster = 0; icluster < inputClustersArray->size(); ++icluster) {
+      // Cluster& cluster = *static_cast<Cluster*>(inputClustersArray->At(icluster));
+      const auto& cluster = (*inputClustersArray)[icluster];
```

Comment:
Can this be changed to a range based loop?

Reply:
this is not so easy since `icluster` is used non-trivially in the code.